diff --git a/chrome/browser/sessions/session_restore.cc b/chrome/browser/sessions/session_restore.cc
index 4d0dab3d32682dff9883eb29fa20cc21ead9de2b..93476e4f67a67cc0df831ea5eb9a3070da8d1f6f 100644
--- a/chrome/browser/sessions/session_restore.cc
+++ b/chrome/browser/sessions/session_restore.cc
@@ -1227,6 +1227,7 @@ class SessionRestoreImpl : public BrowserCollectionObserver {
                                : WindowOpenDisposition::NEW_BACKGROUND_TAB;
       params.tabstrip_add_types = add_types;
       is_first_tab = false;
+      BraveModifyStartupTabNavigationParams(startup_tab, browser, params);
       Navigate(&params);
     }
   }
