diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index eadb7349ccad032d36c98e2b252d7f96108a0912..e2764f55130542605fe9039f344beef9c0cfa437 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2640,6 +2640,7 @@ static_library("browser") {
     "//chrome/browser/sharing/glic_experimental_triggering:impl",
     "//chrome/browser/ui/webauthn:impl",
   ]
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (enable_webui_ntp) {
     deps += [
@@ -5352,6 +5353,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
