diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 5fd4058ff2a76b3c057c18ed8bb9244500fd8c70..5a0f56de5cd70e213eb7a0f393e781c2a7fe1d77 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4677,6 +4677,7 @@ const FeatureEntry::FeatureVariation kCanvasAccessibilityChoices[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
