diff --git a/chrome/browser/ui/browser_window/internal/browser_window_features.cc b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
index fc77781ca5f2b80bdae42c555bc5d46e02d38134..453d01583a2c20dcab9901ed2483e4ed309c5cf0 100644
--- a/chrome/browser/ui/browser_window/internal/browser_window_features.cc
+++ b/chrome/browser/ui/browser_window/internal/browser_window_features.cc
@@ -267,7 +267,7 @@ void BrowserWindowFeatures::Init(BrowserWindowInterface* browser) {
           *browser, fullscreen_controller_.get(), app_browser_controller_.get(),
           browser->GetType(),
           browser->GetBrowserForMigrationOnly()->is_trusted_source(),
-          browser->GetUnownedUserDataHost());
+          browser->GetUnownedUserDataHost(), *browser);
 
   immersive_mode_controller_ =
       GetUserDataFactory()
