diff --git a/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc b/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
index 522adcc2c4d0428ab17d3a1d93b6e1cc84a374ae..dc2028bab979a48b6d0b6b4c640a4f14411b4175 100644
--- a/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
+++ b/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
@@ -697,6 +697,7 @@ void DownloadToolbarUIController::UpdateIcon() {
   }
   action_item_->SetProperty(kActionItemUnderlineIndicatorKey, is_icon_active);
 
+  icon_color = GetIconColor(state_, active_, browser_view_->GetColorProvider());
   action_item_->SetImage(ui::ImageModel::FromVectorIcon(*new_icon, icon_color));
 
   // Update the toolbar button's tooltip.
@@ -746,6 +747,7 @@ void DownloadToolbarUIController::UpdateIcon() {
     return;
   }
   progress_ring->SetDownloading(progress_info_.progress_percentage);
+  UpdateIcon_BraveImpl(browser_view_, action_item_);
 }
 
 void DownloadToolbarUIController::OpenPrimaryDialog() {
