diff --git a/chrome/browser/ui/page_action/page_action_controller.cc b/chrome/browser/ui/page_action/page_action_controller.cc
index f039fe635e4a424db5d61616ff1fa787658a4cd0..d4411ad5fc995e852924da9a59a7bc6f8db9a0d8 100644
--- a/chrome/browser/ui/page_action/page_action_controller.cc
+++ b/chrome/browser/ui/page_action/page_action_controller.cc
@@ -69,6 +69,7 @@ void ScopedPageActionActivity::RegisterWillDestroyControllerCallback() {
   }
 }
 
+namespace chromium_impl {
 PageActionControllerImpl::PageActionControllerImpl(
     PinnedToolbarActionsModel* pinned_actions_model,
     PageActionModelFactory* page_action_model_factory,
@@ -476,7 +477,7 @@ std::unique_ptr<PageActionModelInterface> PageActionControllerImpl::CreateModel(
   if (page_action_model_factory_ != nullptr) {
     return page_action_model_factory_->Create(action_id, is_ephemeral);
   } else {
-    return std::make_unique<PageActionModel>(action_id, is_ephemeral);
+    return std::make_unique<::page_actions::PageActionModel>(action_id, is_ephemeral);
   }
 }
 
@@ -546,6 +547,7 @@ void PageActionControllerImpl::OnIsChipShowingChanged(
                                                   is_chip_showing);
 }
 
+}  // namespace chromium_impl
 std::ostream& operator<<(std::ostream& os, const SuggestionChipConfig& config) {
   os << "{ should_animate: " << config.should_animate
      << ", should_announce_chip: " << config.should_announce_chip << " }";
