diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 7fda559df8b27e3d558a78b8aebb4b4ad0fef7b4..9f87b38c62f437779d46187d51092f8a1a9cf62a 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3816,6 +3816,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
