diff --git a/content/browser/renderer_host/navigation_request.cc b/content/browser/renderer_host/navigation_request.cc
index c79bb9e184312a5e3141dc8859c55bacd906902e..2d81f367e34cbe4cde9417ad066f2a0e398f4fed 100644
--- a/content/browser/renderer_host/navigation_request.cc
+++ b/content/browser/renderer_host/navigation_request.cc
@@ -3930,6 +3930,9 @@ void NavigationRequest::OnRequestRedirected(
   common_params_->url = redirect_info.new_url;
   common_params_->method = redirect_info.new_method;
   common_params_->referrer->url = GURL(redirect_info.new_referrer);
+  GetContentClient()->browser()->MaybeHideReferrer(
+      browser_context, common_params_->url,
+      GetTopDocumentGURL(frame_tree_node_), &common_params_->referrer);
   common_params_->referrer = Referrer::SanitizeForRequest(
       common_params_->url, *common_params_->referrer);
 
@@ -5851,6 +5854,10 @@ void NavigationRequest::OnStartChecksComplete(
     begin_params_->headers = headers.ToString();
   }
 
+  GetContentClient()->browser()->MaybeHideReferrer(
+      frame_tree_node_->navigator().controller().GetBrowserContext(),
+      common_params_->url, GetTopDocumentGURL(frame_tree_node_),
+      &common_params_->referrer);
   // TODO(clamy): Avoid cloning the navigation params and create the
   // ResourceRequest directly here.
   std::vector<std::unique_ptr<NavigationLoaderInterceptor>> interceptor;
