diff --git a/DEPS b/DEPS
index 181cf708cb1dbd4fd3bef9a09b568f418b4ab7da..27f1542a97b2bb3eb0b937c3bd66fd97c727daec 100644
--- a/DEPS
+++ b/DEPS
@@ -4357,7 +4357,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
