diff --git a/net/BUILD.gn b/net/BUILD.gn
index a8538fca26747af78245e983cb6fb661d950239b..69473395bb02f32af0730bee31051939327f9d8a 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1300,6 +1300,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
