diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index c35a1a50221e367c8b5a9483a75ebee5bf4cb18a..7a0a77a64180caf7037b916340906b7a9bbbe3f4 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -164,6 +164,7 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/searchbox:build_ts",
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
+    "//chrome/browser/resources/settings:build_ts", "//brave/ui/webui/resources:build_ts"
   ]
   tsconfig_base = "tsconfig_base.json"
   composite = true
@@ -248,6 +249,7 @@ ts_library("build_ts") {
             "${node_modules}/@lit/reactive-element/reactive-element.d.ts",
             target_gen_dir),
   ]
+  import("//brave/third_party/lit/v3_0/sources.gni") visibility += brave_lit_build_ts_visibility
 }
 
 bundle_js("build_bundle") {
