diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 7080534e60a10cfe2325420034750e32145d81b7..07945f9e1c6d0f925aa9895e5264913cd5235443 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2814,6 +2814,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
